
 *** Reading local file: /Users/savithasathyanathan/airflow/logs/analyzeLog/print_log_error_aapl/2021-03-05T00:00:00+00:00/1.log
[2021-03-13 20:44:05,710] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: analyzeLog.print_log_error_aapl 2021-03-05T00:00:00+00:00 [queued]>
[2021-03-13 20:44:05,722] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: analyzeLog.print_log_error_aapl 2021-03-05T00:00:00+00:00 [queued]>
[2021-03-13 20:44:05,722] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-03-13 20:44:05,722] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-03-13 20:44:05,722] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-03-13 20:44:05,733] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): print_log_error_aapl> on 2021-03-05T00:00:00+00:00
[2021-03-13 20:44:05,736] {standard_task_runner.py:52} INFO - Started process 7597 to run task
[2021-03-13 20:44:05,744] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'analyzeLog', 'print_log_error_aapl', '2021-03-05T00:00:00+00:00', '--job-id', '438', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/airFlowLogAnalyzer.py', '--cfg-path', '/var/folders/nq/nrqknm2s1pz7hdm2m6mblyfm0000gn/T/tmpo365_2wi', '--error-file', '/var/folders/nq/nrqknm2s1pz7hdm2m6mblyfm0000gn/T/tmpr01505nb']
[2021-03-13 20:44:05,746] {standard_task_runner.py:77} INFO - Job 438: Subtask print_log_error_aapl
[2021-03-13 20:44:05,787] {logging_mixin.py:104} INFO - Running <TaskInstance: analyzeLog.print_log_error_aapl 2021-03-05T00:00:00+00:00 [running]> on host savithas-air.lan
[2021-03-13 20:44:05,831] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=analyzeLog
AIRFLOW_CTX_TASK_ID=print_log_error_aapl
AIRFLOW_CTX_EXECUTION_DATE=2021-03-05T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-03-05T00:00:00+00:00
[2021-03-13 20:44:05,831] {bash.py:135} INFO - Tmp dir root location: 
 /var/folders/nq/nrqknm2s1pz7hdm2m6mblyfm0000gn/T
[2021-03-13 20:44:05,832] {bash.py:158} INFO - Running command: 
  echo -e "total error count:" 102 "
List of errors for APPLE:" ['[2021-03-06 22:19:26,475] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:01:52,479] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-07 00:26:13,636] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:59:08,749] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:04:12,690] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:52:30,608] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:19:31,081] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:02:03,892] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:19:35,767] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:02:15,086] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:05:28,434] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:19:03,978] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:00:17,738] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:54:52,396] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:00:07,022] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:49:48,965] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-07 00:22:14,183] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:19:21,542] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:01:38,156] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-07 00:22:23,788] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:05:32,651] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:19:10,770] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:00:21,918] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-07 09:56:18,975] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-07 14:47:03,674] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-07 14:39:43,771] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-07 14:34:40,820] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:19:15,884] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:01:18,290] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:19:40,208] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:02:26,044] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:56:08,170] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:01:31,279] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:51:05,151] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:55:09,313] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:00:13,569] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:50:06,931] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:59:04,545] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:04:08,034] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:52:25,583] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:20:32,453] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:05:14,819] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 23:29:12,399] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:21:01,863] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:59:00,347] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:04:03,345] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:52:15,516] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-07 10:06:36,542] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:20:27,271] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:05:03,858] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:21:06,758] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:55:20,539] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:00:28,423] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:50:18,127] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 23:31:52,069] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:27:01,882] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:31:54,021] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:38:55,805] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 23:16:46,207] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 23:23:15,061] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-07 00:22:09,817] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:55:45,568] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:00:49,732] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:50:43,085] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:56:19,263] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:01:47,927] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:51:16,371] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:19:45,583] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:02:34,740] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:55:56,750] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:01:13,982] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:50:54,454] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:58:56,065] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:03:58,673] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:52:02,249] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:54:57,198] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:00:23,956] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:49:55,063] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:20:22,312] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:04:53,039] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:20:37,555] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:05:23,959] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:54:47,566] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:59:49,649] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:49:44,602] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:58:51,624] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:03:53,937] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:51:46,050] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:20:42,142] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:05:37,166] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:19:54,924] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:04:42,159] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-07 10:03:50,627] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:58:42,600] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:03:45,668] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:51:30,229] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:55:32,960] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:00:35,044] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 21:50:30,742] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:19:50,398] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 22:02:39,242] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n', '[2021-03-06 23:41:53,890] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.\n'] 
[2021-03-13 20:44:05,840] {bash.py:169} INFO - Output:

[2021-03-13 20:44:05,845] {bash.py:173} INFO - total error count: 102
[2021-03-13 20:44:05,845] {bash.py:173} INFO - List of errors for APPLE: [[2021-03-06 22:19:26,475] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,846] {bash.py:173} INFO - , [2021-03-06 22:01:52,479] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,846] {bash.py:173} INFO - , [2021-03-07 00:26:13,636] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,846] {bash.py:173} INFO - , [2021-03-06 21:59:08,749] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,847] {bash.py:173} INFO - , [2021-03-06 22:04:12,690] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,847] {bash.py:173} INFO - , [2021-03-06 21:52:30,608] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,847] {bash.py:173} INFO - , [2021-03-06 22:19:31,081] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,848] {bash.py:173} INFO - , [2021-03-06 22:02:03,892] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,848] {bash.py:173} INFO - , [2021-03-06 22:19:35,767] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,848] {bash.py:173} INFO - , [2021-03-06 22:02:15,086] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,849] {bash.py:173} INFO - , [2021-03-06 22:05:28,434] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,849] {bash.py:173} INFO - , [2021-03-06 22:19:03,978] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,849] {bash.py:173} INFO - , [2021-03-06 22:00:17,738] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,849] {bash.py:173} INFO - , [2021-03-06 21:54:52,396] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,850] {bash.py:173} INFO - , [2021-03-06 22:00:07,022] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,850] {bash.py:173} INFO - , [2021-03-06 21:49:48,965] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,850] {bash.py:173} INFO - , [2021-03-07 00:22:14,183] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,851] {bash.py:173} INFO - , [2021-03-06 22:19:21,542] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,851] {bash.py:173} INFO - , [2021-03-06 22:01:38,156] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,851] {bash.py:173} INFO - , [2021-03-07 00:22:23,788] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,852] {bash.py:173} INFO - , [2021-03-06 22:05:32,651] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,852] {bash.py:173} INFO - , [2021-03-06 22:19:10,770] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,852] {bash.py:173} INFO - , [2021-03-06 22:00:21,918] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,853] {bash.py:173} INFO - , [2021-03-07 09:56:18,975] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,853] {bash.py:173} INFO - , [2021-03-07 14:47:03,674] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,853] {bash.py:173} INFO - , [2021-03-07 14:39:43,771] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,854] {bash.py:173} INFO - , [2021-03-07 14:34:40,820] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,854] {bash.py:173} INFO - , [2021-03-06 22:19:15,884] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,854] {bash.py:173} INFO - , [2021-03-06 22:01:18,290] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,855] {bash.py:173} INFO - , [2021-03-06 22:19:40,208] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,855] {bash.py:173} INFO - , [2021-03-06 22:02:26,044] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,855] {bash.py:173} INFO - , [2021-03-06 21:56:08,170] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,856] {bash.py:173} INFO - , [2021-03-06 22:01:31,279] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,856] {bash.py:173} INFO - , [2021-03-06 21:51:05,151] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,856] {bash.py:173} INFO - , [2021-03-06 21:55:09,313] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,857] {bash.py:173} INFO - , [2021-03-06 22:00:13,569] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,857] {bash.py:173} INFO - , [2021-03-06 21:50:06,931] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,858] {bash.py:173} INFO - , [2021-03-06 21:59:04,545] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,858] {bash.py:173} INFO - , [2021-03-06 22:04:08,034] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,858] {bash.py:173} INFO - , [2021-03-06 21:52:25,583] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,859] {bash.py:173} INFO - , [2021-03-06 22:20:32,453] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,859] {bash.py:173} INFO - , [2021-03-06 22:05:14,819] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,859] {bash.py:173} INFO - , [2021-03-06 23:29:12,399] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,860] {bash.py:173} INFO - , [2021-03-06 22:21:01,863] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,860] {bash.py:173} INFO - , [2021-03-06 21:59:00,347] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,860] {bash.py:173} INFO - , [2021-03-06 22:04:03,345] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,861] {bash.py:173} INFO - , [2021-03-06 21:52:15,516] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,861] {bash.py:173} INFO - , [2021-03-07 10:06:36,542] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,862] {bash.py:173} INFO - , [2021-03-06 22:20:27,271] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,862] {bash.py:173} INFO - , [2021-03-06 22:05:03,858] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,862] {bash.py:173} INFO - , [2021-03-06 22:21:06,758] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,863] {bash.py:173} INFO - , [2021-03-06 21:55:20,539] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,863] {bash.py:173} INFO - , [2021-03-06 22:00:28,423] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,864] {bash.py:173} INFO - , [2021-03-06 21:50:18,127] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,865] {bash.py:173} INFO - , [2021-03-06 23:31:52,069] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,866] {bash.py:173} INFO - , [2021-03-06 22:27:01,882] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,866] {bash.py:173} INFO - , [2021-03-06 22:31:54,021] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,866] {bash.py:173} INFO - , [2021-03-06 22:38:55,805] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,867] {bash.py:173} INFO - , [2021-03-06 23:16:46,207] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,867] {bash.py:173} INFO - , [2021-03-06 23:23:15,061] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,867] {bash.py:173} INFO - , [2021-03-07 00:22:09,817] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,868] {bash.py:173} INFO - , [2021-03-06 21:55:45,568] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,868] {bash.py:173} INFO - , [2021-03-06 22:00:49,732] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,868] {bash.py:173} INFO - , [2021-03-06 21:50:43,085] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,869] {bash.py:173} INFO - , [2021-03-06 21:56:19,263] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,869] {bash.py:173} INFO - , [2021-03-06 22:01:47,927] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,869] {bash.py:173} INFO - , [2021-03-06 21:51:16,371] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,870] {bash.py:173} INFO - , [2021-03-06 22:19:45,583] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,870] {bash.py:173} INFO - , [2021-03-06 22:02:34,740] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,870] {bash.py:173} INFO - , [2021-03-06 21:55:56,750] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,871] {bash.py:173} INFO - , [2021-03-06 22:01:13,982] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,871] {bash.py:173} INFO - , [2021-03-06 21:50:54,454] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,871] {bash.py:173} INFO - , [2021-03-06 21:58:56,065] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,871] {bash.py:173} INFO - , [2021-03-06 22:03:58,673] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,872] {bash.py:173} INFO - , [2021-03-06 21:52:02,249] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,872] {bash.py:173} INFO - , [2021-03-06 21:54:57,198] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,872] {bash.py:173} INFO - , [2021-03-06 22:00:23,956] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,873] {bash.py:173} INFO - , [2021-03-06 21:49:55,063] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,873] {bash.py:173} INFO - , [2021-03-06 22:20:22,312] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,873] {bash.py:173} INFO - , [2021-03-06 22:04:53,039] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,874] {bash.py:173} INFO - , [2021-03-06 22:20:37,555] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,874] {bash.py:173} INFO - , [2021-03-06 22:05:23,959] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,874] {bash.py:173} INFO - , [2021-03-06 21:54:47,566] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,875] {bash.py:173} INFO - , [2021-03-06 21:59:49,649] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,875] {bash.py:173} INFO - , [2021-03-06 21:49:44,602] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,875] {bash.py:173} INFO - , [2021-03-06 21:58:51,624] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,876] {bash.py:173} INFO - , [2021-03-06 22:03:53,937] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,876] {bash.py:173} INFO - , [2021-03-06 21:51:46,050] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,876] {bash.py:173} INFO - , [2021-03-06 22:20:42,142] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,877] {bash.py:173} INFO - , [2021-03-06 22:05:37,166] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,877] {bash.py:173} INFO - , [2021-03-06 22:19:54,924] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,878] {bash.py:173} INFO - , [2021-03-06 22:04:42,159] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,879] {bash.py:173} INFO - , [2021-03-07 10:03:50,627] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,880] {bash.py:173} INFO - , [2021-03-06 21:58:42,600] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,880] {bash.py:173} INFO - , [2021-03-06 22:03:45,668] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,880] {bash.py:173} INFO - , [2021-03-06 21:51:30,229] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,881] {bash.py:173} INFO - , [2021-03-06 21:55:32,960] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,881] {bash.py:173} INFO - , [2021-03-06 22:00:35,044] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,881] {bash.py:173} INFO - , [2021-03-06 21:50:30,742] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,882] {bash.py:173} INFO - , [2021-03-06 22:19:50,398] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,882] {bash.py:173} INFO - , [2021-03-06 22:02:39,242] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,883] {bash.py:173} INFO - , [2021-03-06 23:41:53,890] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-03-13 20:44:05,883] {bash.py:173} INFO - ]

[2021-03-13 20:44:05,883] {bash.py:177} INFO - Command exited with return code 0
[2021-03-13 20:44:05,901] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=analyzeLog, task_id=print_log_error_aapl, execution_date=20210305T000000, start_date=20210314T044405, end_date=20210314T044405
[2021-03-13 20:44:05,918] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-03-13 20:44:05,962] {local_task_job.py:146} INFO - Task exited with return code 0


Version: v2.0.1
Git Version:.release:2.0.1+beb8af5ac6c438c29e2c186145115fb1334a3735